src/table/create.rs

```diff
@@ -84,6 +85,7 @@ pub struct TableCreateStatement {
     pub(crate) indexes: Vec<IndexCreateStatement>,
     pub(crate) foreign_keys: Vec<ForeignKeyCreateStatement>,
     pub(crate) if_not_exists: bool,
+    pub(crate) check: Option<SimpleExpr>,
```

Comment:
Can we allow multiple check constraint at table level?

```suggestion
    pub(crate) checks: Vec<SimpleExpr>,
```